utils/http/errors.py

```diff
@@ -0,0 +1,12 @@
+class ResponseError(Exception):
+    """Generic ResponseError exception
+
+    It helps to stop processing a view function by raising an exception.
+    It is like `Http404` exception but with ability to define your response.
+
+    NOTE: `htk.middleware.classes.CatchRaisedExceptionResponseMiddleware` MUST
+          be in MIDDLEWARES in Django Settings.
+    """
+
+    def __init__(self, response):
+        self.response = response
```

Comment:
I think it should have `response` as well as `status_code` as a parameter.

Reply:
Done

Reply:
Does `response` always have a `status_code` set or not?

Reply:
`response` must always have a `status_code` and it defaults to `200` if not set.

I changed structure to accept also `string` and update status code if it is `200` because it does not make sense to have status code 200 if this is an raised response error.